app/src/main/java/com/powerhungers/fomezero/common/exception/DomainException.kt

```diff
@@ -9,4 +9,5 @@ sealed class ParamException(message: String, title: String? = null) :
     DomainException(message, title)
 
 class EmptyEmailException : ParamException(Constants.EMPTY_EMAIL)
-class EmptyPasswordException : ParamException(Constants.EMPTY_PASSWORD)
\ No newline at end of file
+class EmptyPasswordException : ParamException(Constants.EMPTY_PASSWORD)
+class MissInformationException : ParamException(Constants.MISS_INFOR)
```

Comment:
```suggestion
class MissInformationException : ParamException(Constants.MISSING_INFORMATION)
```
Não tem necessidade ficar abreviando esse tipo de coisa, quando mais claro e mais objetivo melhor